contracts/NFPlant.sol

```diff
@@ -24,40 +35,48 @@ contract LanVar is ERC721, ERC721Enumerable, ERC721URIStorage, Ownable, Chainlin
         address owner;
     }
 
-    mapping (bytes32 => requestInfo) private requests;
+    mapping(bytes32 => requestInfo) private requests;
 
-    IERC20 private token =  IERC20(0x2C8f56E5f468E1708555A9B334D94973509778E2);
-    uint256 private rewardAmount = 1 * 10 ** 18;
+    IERC20 private token = IERC20(0x2C8f56E5f468E1708555A9B334D94973509778E2);
+    uint256 private rewardAmount = 1 * 10**18;
 
     address private oracle;
     bytes32 private jobId;
     uint256 private fee;
 
-    constructor() ERC721("LanVar", "LVN") {
+    constructor() ERC721("NFPlant", "LVN") {
```

Comment:
should change LVN -> NFP

Reply:
Updated in https://github.com/viefam/metabio-smart-contracts/pull/14/commits/b7e88ece47ba4eb8b77133e4cb65c66b6a9da9a1